components/admin_console/team_channel_settings/channel/details/channel_details.jsx

```diff
@@ -56,10 +57,14 @@ export default class ChannelDetails extends React.Component {
         };
     }
 
-    componentDidUpdate(prevProps) { // TODO: find out how to do this without the lifecycle
-        if (prevProps.totalGroups !== this.props.totalGroups) {
+    componentDidUpdate(prevProps) {
+        if (this.props.channel !== prevProps.channel) {
```

Comment:
You may have already, but I would just verify that this conditional isn't true every time (bearing in mind I didn't check the source of this channel prop).